Ничего не идёт по плану. Ничего.

UniverseGuard opened the pull request.

UniverseGuard committed:
Обновление translations/texts/codex/apex/apexspace1.codex.json

UniverseGuard committed:
Обновление translations/texts/objects/space/telescope/telescope.object.json

UniverseGuard committed:
Обновление translations/texts/objects/ancient/ancientmural/shared_ancientmural/shared_ancientmural.json

UniverseGuard committed:
Обновление translations/texts/objects/ancient/shared_monolithstatue/shared_monolithstatue.json

UniverseGuard committed:
Обновление translations/texts/objects/ancient/shared_ancientpot/shared_ancientpot.json

UniverseGuard committed:
Обновление translations/texts/objects/ancient/monolithgate/monolithgate.object.json

UniverseGuard committed:
Обновление translations/texts/npcs/crew/crewmembermechanic.npctype.json

UniverseGuard committed:
Обновление translations/texts/npcs/crew/crewmemberjanitor.npctype.json

UniverseGuard committed:
Обновление translations/texts/npcs/crew/crewmemberengineer.npctype.json

UniverseGuard committed:
Обновление translations/texts/npcs/tenants/evilglitchtenant.npctype.json

UniverseGuard committed:
Обновление translations/texts/npcs/tenants/glitchcastle.npctype.json

UniverseGuard committed:
Обновление translations/texts/npcs/mission/penguinscientist.npctype.json

UniverseGuard committed:
Обновление translations/texts/objects/generic/arcadegame/shared_arcadegame.json

UniverseGuard committed:
Обновление translations/texts/objects/generic/arcadegame/shared_arcadegame.json

UniverseGuard committed:
Обновление translations/texts/objects/generic/arcadegame/shared_arcadegame.json

UniverseGuard committed:
Обновление translations/texts/objects/space/stationdoor/shared_stationdoor.json

UniverseGuard committed:
Обновление translations/texts/objects/ancient/ancientbust/ancientbust.object.json

Xorras commented:
Не бойся менять построения предложений, плёткой за такое бить не будем... (пока... хе-хе)
Главное чтобы смысл оставался тот же.

UniverseGuard committed:
Обновление translations/texts/objects/ancient/ancientbust/ancientbust.object.json

UniverseGuard committed:
Обновление translations/texts/objects/actionfigure/parasprite/paraspriteaf.object.json

UniverseGuard committed:
Merge 5fd0461df0a30bbb77e7f50ac79842be1e06b2dd into web-interface

UniverseGuard committed:
Обновление translations/texts/objects/actionfigure/parasprite/paraspriteaf.object.json

UniverseGuard committed:
Update crewmemberjanitor.npctype.json

UniverseGuard committed:
Update shared_ancientmural.json

UniverseGuard committed:
Update shared_arcadegame.json

UniverseGuard committed:
Update telescope.object.json

Xorras commented on translations/texts/objects/actionfigure/parasprite/paraspriteaf.object.json:
```diff
@@ -20,7 +20,7 @@
     "Texts": {
       "Eng": "The label reads, \"A braindead bioweapon created by irresponsible human scientists.\"",
-      "Rus": ""
+      "Rus": "Надпись гласит: \"Безмозглое биооружие, созданное безответсвенными учеными людей\"."
```
"безответсТвенными"

И скорее не "учеными людей", а "человеческими учёными" больше подходит

UniverseGuard replied:
Я оба варианта рассматривал, но ни первый, ни второй особо не понравились. Вообще было бы неплохо заменить как-то, но не суть.

UniverseGuard committed:
Update paraspriteaf.object.json

UniverseGuard committed:
Обновление translations/texts/objects/apex/apexbustmale/apexbustmale.object.json

UniverseGuard committed:
Обновление translations/texts/objects/apex/apexconsole1/apexconsole1.object.json

UniverseGuard committed:
Обновление translations/texts/objects/apex/apexradio/apexradio.object.json

UniverseGuard committed:
Merge branch 'web-interface' into web-interface

UniverseGuard committed:
Обновление translations/texts/items/active/weapons/melee/abilities/spear/elementalspin/elementalspin.weaponability.json

UniverseGuard committed:
Обновление translations/texts/items/active/weapons/other/adaptablecrossbow/adaptableammo.weaponability.json

UniverseGuard committed:
Обновление translations/texts/items/active/weapons/staff/abilities/guidedbolt/guidedbolt.weaponability.json

UniverseGuard committed:
Обновление translations/texts/items/active/weapons/staff/abilities/effectzone/healingzone.weaponability.json

UniverseGuard committed:
Обновление translations/texts/items/active/weapons/staff/abilities/effectzone/slowzone.weaponability.json

UniverseGuard committed:
Обновление translations/texts/items/active/weapons/staff/abilities/effectzone/lowgravzone.weaponability.json

UniverseGuard committed:
Обновление translations/texts/dialog/alpaca.config.json

UniverseGuard committed:
Обновление translations/texts/dialog/converse.config.json

UniverseGuard commented:
Updated

UniverseGuard committed:
Обновление translations/texts/dialog/converse.config.json

UniverseGuard committed:
Обновление translations/texts/dialog/converse.config.json

UniverseGuard committed:
Update crewmembermechanic.npctype.json

UniverseGuard commented on translations/texts/npcs/crew/crewmembermechanic.npctype.json:
```diff
@@ -8,7 +8,8 @@
       "Eng": "I'll keep clearing out the carborator while I'm here. Fuel efficiency is up 10%!",
-      "Rus": "Я буду чистить карбюратор пока я здесь. Эффективность топлива увеличена на 10%!"
+      "Rus": "Я продолжаю чистить карбюратор, пока я здесь. Эффективность потребления топлива выросла на 10%!"
+
```
![image](https://user-images.githubusercontent.com/30653739/29250890-5dfbbaf2-8053-11e7-987e-e9b9918575f1.png)
Тут явно что-то не так :/

UniverseGuard committed:
Обновление translations/texts/codex/apex/apexspace1.codex.json

UniverseGuard committed:
Обновление translations/texts/codex/human/humanspace1.codex.json

UniverseGuard committed:
Merge branch 'web-interface' into web-interface

UniverseGuard committed:
Update converse.config.json

xomachine commented on translations/texts/dialog/converse.config.json:
```diff
@@ -17060,7 +17061,7 @@
     "Texts": {
       "Eng": "What kind of planet was Earth? Oh... Is it too soon to ask this question? My bad.",
-      "Rus": "Какой была Земля? О... наверное, пока не следует задавать таких вопросов. Прости."
+      "Rus": "Какого вида была Земля? Ой... Слишком рано задавать этот вопрос? Мой вина."
```
Что значит "какого вида"? Разве тут речь идёт о виде земли?

UniverseGuard replied:
В самом старбаунде планеты распледеляются по типам. Вид Земли и означает вид планеты. Писать вид планеты Земля - это то же, что писать центр города, т.к город и означает центр... Ну мне так-то все равно, меня не устроила вторая половина его перевода

xomachine commented on translations/texts/dialog/converse.config.json:
```diff
@@ -17075,7 +17076,7 @@
     "Texts": {
       "Eng": "What kind of plant did you grow from?",
-      "Rus": "Из какого растения вы вырастаете?"
+      "Rus": "Из какого вида растений ты сделан?"
```
На мой взгляд, стало только хуже.

UniverseGuard replied:
Ну расти из растений тоже такое себе

UniverseGuard replied:
Да и "вы" тут не очень уместно

xomachine commented on translations/texts/dialog/converse.config.json:
```diff
@@ -17046,7 +17047,7 @@
     "Texts": {
       "Eng": "What is your business here?",
-      "Rus": "Чем ты здесь занимаешься?"
+      "Rus": "Какое твоё дело здесь?"
```
Мне кажется, или в русском языке подобный перевод несёт негативный оттенок, которого нет в оригинале?

UniverseGuard replied:
Да в русском языке и словом "Уважаемый" можно оскорбить :D Это странно немного, но каждый может воспринимать по-своему

UniverseGuard committed:
Update converse.config.json

UniverseGuard committed:
Обновление translations/texts/species/humanfemalenamegen.config.json

UniverseGuard committed:
Обновление translations/texts/species/humanfemalenamegen.config.json

UniverseGuard committed:
Почему никто этого не переводил?

UniverseGuard committed:
Обновление translations/texts/dialog/converse.config.json

UniverseGuard committed:
Обновление translations/texts/interface/windowconfig/keybindingsmenu.config.json

UniverseGuard committed:
Update paraspriteaf.object.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Merge branch 'web-interface' into web-interface

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/codex/hylotl/hylotlspace1.codex.json

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -338,7 +340,7 @@
     "Texts": {
       "Eng": "\"Poor lonely octopus! I have a song in mind just for this.\" she says.",
-      "Rus": ""
+      "Rus": "\"Бедный одинокий осьминог! У меня есть одна песня в голове насчёт этого.\" - сказала она."
```
Лучше сказать что песня вертится в голове, а то "есть" как-то не по русски звучит.

xomachine replied:
всё еще актуально
(вертится вместо есть)

UniverseGuard committed:
Merge 38611a83383bfdc83760f1a43c80cdd6e5a7ac4b into web-interface

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -350,7 +352,7 @@
     "Texts": {
       "Eng": "\"Sad old bear, destined to roam, after the banks, repossessed his home.\"",
-      "Rus": ""
+      "Rus": "\"Старый грустный медведь, предопределённый скитаниям, после того, как банки изъяли за неуплату его дом. \""
```
Этот перевод сложнее чем обычные. Лучше сохранить все рифмы (это же песня!)

UniverseGuard replied:
Сначала я даже не замечал рифмы :/ Потом исправлял. Всё исправим, partner.

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -374,7 +376,7 @@
     "Texts": {
       "Eng": "\"So tell me,\" she begins, \" Why are you interested in this unopened library, of all places?\"",
-      "Rus": ""
+      "Rus": "\"Итак, расскажи мне,\" - она начала - \"Почему ты заинтересован именно этой неоткрытой библиотеке из всех мест?\""
```
Рассогласование падежей

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -458,7 +460,7 @@
     "Texts": {
       "Eng": "\"That's a bit weird, we don't know either other that well!\" she responds. I barely have time to process my rejection as I feel the familiar tug of time, and everything fades.",
-      "Rus": ""
+      "Rus": "\"Это немного странно, мы недостаточно хорошо знаем друг друга!\" - отвечала она. У меня едва хватило времени, чтобы обработать отказ, так как я почувствовал знакомое прикосновение времени, и всё растворилось."
```
"обработать" не особо похоже на подходящее слово в данной реплике.

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -518,7 +520,7 @@
     "Texts": {
       "Eng": "\"We're all moving through time, at a rate of around 1 second per second.\" she elaborates.",
-      "Rus": ""
+      "Rus": "\"Мы все двигаемся во времени, со скоростью около 1 секунды в секунду.\" - она заключила."
```
Думаю всё же стоит читать в слух пару раз то, что переводишь. Тут явно надо поменять местами "она" и "заключила"

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -542,7 +544,7 @@
     "Texts": {
       "Eng": "\"Well of course it isn't - I was interrupted by you flailing about and screaming for help.\"",
-      "Rus": ""
+      "Rus": "\"Ну, конечно же это не так, меня прервали твои размахивания и крики о помощи.\""
```
Что за "размахивания"? Кто так говорит в реальной жизни?

UniverseGuard replied:
@xomachine Я

UniverseGuard replied:
@xomachine И не забудь, что это даже не реальная жизнь, а просто новелла, по типу бесконечного лета :D

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -494,7 +496,7 @@
     "Texts": {
       "Eng": "\"The ceremony!\" she exclaims. \"But it is a ceremony of tradition, not divine power. And it is not even finished, either.\"",
-      "Rus": ""
+      "Rus": "\"Церемония!\" - она воскликнула - \"Но эта церемония - традиция, не божественная сила. И, в любом случае, она даже не закончена.\""
```
тоже порядок слов. еще я не уверен по поводу "божественной силы". разве может церемония являться божественной силой?

UniverseGuard replied:
Всё возможно. Лучше в голову ничего не пришло. По сути церемония = ритуал (с анг.) Можно сделать, что ритуал - традиция, а не посвященная божеству сила. @xomachine

UniverseGuard replied:
Но это звучит ещё хуже :/ Просто оставлю пустой. Будет что получше - сделаю

xomachine reviewed (commented):
Стихи надо переводить стихами, а то получается бред. И всё же стоит читать свой перевод и по возможности избавляться от косноречий в нём.

UniverseGuard committed:
Update beautifulattempt.sbvn.json

UniverseGuard committed:
Update beautifulattempt.sbvn.json

UniverseGuard committed:
Update beautifulattempt.sbvn.json

UniverseGuard committed:
Update beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/objects/hylotl/airlockdoor/airlockdoor.object.json

UniverseGuard committed:
Merge branch 'web-interface' into web-interface

UniverseGuard commented:
С днём рождения меня 🎂

UniverseGuard committed:
Merge branch 'web-interface' into web-interface

UniverseGuard commented:
@Xorras @xomachine В окно описания предметов влезает до 27 символов ведь?

Xorras commented:
Лимит касается только названия. Описание надо определить еще...

Tvirian commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -386,7 +388,7 @@
     "Texts": {
       "Eng": "\"Starfish lovers, oh so sweet, how do you dance, without some feet?\"",
-      "Rus": ""
+      "Rus": "\"Любители морских звёзд, как же сладки, как вы станцуете без единой ноги?\""
```
Не знаю контекста, но подозреваю, что тут не любители звёзд, а сами морские звёзды. Это же у них нет ножек, чтобы танцевать. Звёзды-любовники, сладкая парочка, иглокожая пара - не стесняйтесь ворочать синонимами и порядком слов. Типа:
Звёзды морские, найдя любви время, как вы танцуете, ног не имея?
Звёзды морские, единое сердце - со встречи за партией в дартс в клубе местном.
Ни или как-то так : )

UniverseGuard replied:
Я вообщем тоже контекста особо не знаю :D , там вся история разбросана по всем 7 страницам.

UniverseGuard replied:
Но это очень интересный кусок игры

Tvirian commented on translations/texts/objects/generic/arcadegame/shared_arcadegame.json:
```diff
@@ -26,7 +26,7 @@
     "Texts": {
       "Eng": "I am not a fan of Mazebound64. I've heard horror stories about Miniknog testing... Mazes give me the creeps.",
-      "Rus": ""
+      "Rus": "Я не фанат Mazebound64. Я наслышан устрашающих история про эксперименты Министерства... У меня от лабиринтов мороз по коже."
```
Тут явно какая-то ошибка. Уж либо "Я наслушался жутких историй про эксперименты..." либо "Я наслышан о кошмарных экспериментах..."
Но первый вариант, мне кажется, уместнее. "Наслышан" - как-то слишком официозно для описания игрового автомата. А "устрашающий", полагаю, совсем не то слово, которым вы бы охарактеризовали в разговоре секретные правительственные эксперименты на людях ; )

UniverseGuard committed:
Update beautifulattempt.sbvn.json

UniverseGuard committed:
Update shared_arcadegame.json

xomachine commented:
@UniverseGuard  не обращай внимания на этот конфликт. Я его разрешу при мерже

UniverseGuard commented:
Если бы не это, то было бы отлично 
![image](https://user-images.githubusercontent.com/30653739/29336768-96af1eaa-8217-11e7-87b2-dbc13ff25b64.png)
.../dialog/converse.config.json

xomachine commented:
эм, возможно это мой косяк. я недавно обновлял интерфейс

мне нужны логи из консоли разработчика

UniverseGuard commented:
Имеешь в виду этой? 
![image](https://user-images.githubusercontent.com/30653739/29336919-2d934486-8218-11e7-9778-c4e6a2e71e06.png)

xomachine commented:
Так, а просто обновить страницу не пробовали? А то я вижу старую версию интерфейса, еще до обновления.

UniverseGuard commented:
После перезагрузки страницы перекинет в /dialog

UniverseGuard commented:
Ничего не идет по плану. Никогда :D

xomachine committed:
Update converse.config.json

Починил (надеюсь)

xomachine commented:
Готово. Надо следить за структурой json, если редактировать напрямую, а не через интерфейс

UniverseGuard commented:
@xomachine Спасибо, обычно просто фиксы мелкие напрямую делаю. Ох уж эти две палочки :D Могут поломать целую страницу

xomachine commented:
Тут лайфхак есть для таких случаев. Если прописывать что-то вроде:
https://sbt-community.github.io/?open=translations/texts/dialog/converse.config.json&goto=kind%20of%20plant то можно перейти сразу к нужному полю (ну или почти к нему, правильный скрол страницы пока выше моих сил)

UniverseGuard committed:
Обновление translations/texts/dialog/converse.config.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Update converse.config.json

UniverseGuard committed:
Обновление translations/texts/dialog/converse.config.json

xomachine commented on translations/texts/dialog/converse.config.json:
```diff
@@ -19245,7 +19245,7 @@
     "Texts": {
       "Eng": "You shine like starlight.",
-      "Rus": ""
+      "Rus": "Ты светишься, словно свечение здвёзд."
```
опечатка

UniverseGuard commented:
Вообщем, пора закончить работу здесь и создать новый пулл, тут слишком много перевода, который конфликтует. И меньше его не становится

UniverseGuard committed:
Update converse.config.json

xomachine committed:
Update glitchcastle.npctype.json

UniverseGuard committed:
Merge branch 'web-interface' into web-interface

UniverseGuard commented:
@xomachine Спасибо за всё, ты реально крут :D Лучшая команда, с которой мне доводилось работать @Xorras          ❤️   ❤️   ❤️

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -21,7 +23,7 @@
     "Texts": {
       "Eng": " I assume that she misunderstood the implication of my remark, but I cannot help stare a little too long into her triumvirate of ruby red eyes.",
-      "Rus": ""
+      "Rus": "Я полагаю, что она не поняла вывод моих замечаний, но я не могу на долго уставиться в её триумвират красных, как рубин, глаз."
```
Звучит как какой-то нонсенс. Либо убрать, либо переделать. Я не могу пока придумать хорошего перевода для этой фразы.

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -33,7 +35,7 @@
     "Texts": {
       "Eng": "\"...I guess the standard of art is different in your time, huh?\" she says flatly. I presume that the picture was not, in fact, a joke.",
-      "Rus": ""
+      "Rus": "\"... Я полагаю, стандарты искусства другие в ваше время, хах?\" - сказала она решительно. Я предполагаю, что это, по факту, была не шутка."
```
на самом деле вместо по факту звучит натуральнее

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -117,7 +121,7 @@
     "Texts": {
       "Eng": "\"But perhaps...\" she says whilst looking intently at the staff in her hand, \"Perhaps finishing the ceremony will help send you back?\"",
-      "Rus": ""
+      "Rus": "\"Но, возможно...\" - сказала она, пока пристально смотрела на жезл в её руке - \"Возможно, завершение церемонии поможет отправить тебя обратно?\""
```
в своей руке

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -226,7 +232,7 @@
     "Texts": {
       "Eng": "\"I have finished!\" she announces. Her smile glitters in the sunset like the lure of a beautiful angler fish.",
-      "Rus": ""
+      "Rus": "\"Я закончила!\" - объявила она. Её улыбка блестит в солнечном свете, как соблазн прекрасного морского чёрта."
```
может всё таки приманка, а не соблазн? у морского черта над головой висит обычно

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -358,7 +364,7 @@
     "Texts": {
       "Eng": "\"Sad old bear, it's all gone wrong, since you realised, you're just a song.\"",
-      "Rus": ""
+      "Rus": "\"Старый грустный медведь, всё пошло неправильно, как только ты понял, что ты просто песня.\""
```
Рифма потеряна

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -430,7 +436,7 @@
     "Texts": {
       "Eng": "\"That is quite tragic! I have a song in mind just for this.\" she says.",
-      "Rus": ""
+      "Rus": "\"Это действеительно печально! У меня в голове вертится олна песня на это счёт.\" - сказала она."
```
действЕительно лучше заменить на весьма

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -853,18 +881,22 @@
     "Texts": {
-      "Eng": "Could she truly love me? Will she be able to come to the present with me? Will that cause a rift in the space-time continuum?"
+      "Eng": "Could she truly love me? Will she be able to come to the present with me? Will that cause a rift in the space-time continuum?",
+      "Rus": "Сможет ли она правда полюбить меня? Будет ли у неё возможность вернуться со мной в настоящее? Не вызовет ли это разлом в космическо-временном континууме?"
```
пространственно-временном

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -1549,18 +1616,22 @@
     "Texts": {
-      "Eng": "She doesn't know what Air Hockey is."
+      "Eng": "She doesn't know what Air Hockey is.",
+      "Rus": "Она не знает что аэрохокей вообще такое."
```
что такое аэрохокей. вообще лишнее

xomachine commented on translations/texts/items/active/weapons/staff/abilities/effectzone/lowgravzone.weaponability.json:
```diff
@@ -7,7 +7,8 @@
     "Texts": {
-      "Eng": "Low Gravity Zone"
+      "Eng": "Low Gravity Zone",
+      "Rus": "Низко гравитационная зона"
```
Низкогравитационная разве не слитно?

UniverseGuard replied:
Ну, в гугле ничего не нашёл, сам гугл говорит раздельно. Уточнять просто негде... Могу просто сделать зону низкой гравитации

xomachine reviewed (commented):
Уф, тут слишком много переводов, чтобы я успел сразу просмотреть все. В общем осталось поправить эти замечания, тогда можно будет мержить на мой взгляд. Если никто другой не имеет ничего против.

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

UniverseGuard committed:
Update beautifulattempt.sbvn.json

UniverseGuard committed:
Update lowgravzone.weaponability.json

UniverseGuard commented:
@xomachine Fixed, над всем остальным буду лучше думать с новым пулл реквестом, который, скорее всего, посвящу этому куску.

UniverseGuard committed:
Merge branch 'web-interface' into web-interface

xomachine committed:
Update converse.config.json

Добавил новый adj3Ending с окончанием ИЙ и немного оживил фразу

xomachine committed:
Update beautifulattempt.sbvn.json

Это и правда звучит странно, пока уберу, до тех пор пока не появится более внятный перевод.

xomachine committed:
Update beautifulattempt.sbvn.json

Убрал нерифмованную фразу из песни.
